ocaml/xapi/xapi_xenops.ml

```diff
@@ -2819,6 +2819,7 @@ let resume ~__context ~self ~start_paused ~force =
        end;
        set_resident_on ~__context ~self;
        Db.VM.set_suspend_VDI ~__context ~self ~value:Ref.null;
+       Xapi_gpumon.clear_vgpu_metadata ~__context ~vm:self;
```

Comment:
Add a comment here saying that this have to happen later in the process to avoid issues